auth/src/main/java/org/openedx/auth/presentation/signin/SignInViewModel.kt

```diff
@@ -52,6 +62,20 @@ class SignInViewModel(
             isMicrosoftAuthEnabled = config.getMicrosoftConfig().isEnabled(),
             isSocialAuthEnabled = config.isSocialAuthEnabled(),
             isLogistrationEnabled = config.isPreLoginExperienceEnabled(),
+            agreement = agreementProvider.getAgreement(isSignIn = true)?.let {
```

Comment:
we can simply use it as a string
```
agreement = agreementProvider.getAgreement(isSignIn = true)

Reply:
I'm making mapping here in the view model